Grammars/Parts/grapart_basic.gap

```diff
@@ -1,27 +1,38 @@
-  struct    = sadd(BASE with unpaired, struct)     |
```

Comment:
to be consistent regarding my old naming scheme, I think it would be more appropriate to rename this file into `gra_cofold_nodangle.gap` and move it up one directory. I don't see an elegant way to separate the `basic` rules from your new cofolding rules, i.e. the grapart_ trick might not work